### DISCUSSION TITLE
Right-Align Modules

### DISCUSSION THREAD

**Discussion by agus-videla** [starship/starship#2403] [Q&A] [answered]
Hi, is there a way to make specific modules appear at the right side of the prompt? For example, I'd like for the current path to appear in the first line on the left (as usual) with cmd_duration module in the same line all the way to the right. Thanks

**Comment by chipbuster** [comment, marked as answer]
It's a popular request. See #498. As of right now we don't have a way to support it.

**Reply by agus-videla** [reply]
I see. Thank you for replying
